src/components/Envelope.vue

```diff
@@ -171,13 +171,14 @@ export default {
 				const recipients = [this.data.to, this.data.cc].flat().map(function(recipient) {
 					return recipient.email
 				})
-				return recipients.length > 0 ? recipients[0] : undefined
+				return recipients.length > 0 ? recipients[0] : 'Blind copy recipients only'
```

Comment:
:-1: I'm fine with this enhancement but 1) it's not translated and 2) this is unrelated to the topic of this PR. Please revert and submit separately :v:

Reply:
ok